recipe/meta.yaml

```diff
@@ -11,38 +11,42 @@ source:
   sha256: 5cac870d6d269cdd71d28def4cae310c609c02aac636c0831176188f16021f44
 
 build:
+  skip: True  # [py<37]
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script:
+    - {{ PYTHON }} -m poetry build
+    - {{ PYTHON }} -m pip install . -vv
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - numpy
   host:
     - numpy >=1.14
     - pip
-    - poetry-core  # >=1.1.0a6
-    - python >=3.7.1,<3.11
+    - poetry-core >=1.1.0a6
```

Comment:
This is correct according to upstream but is failing in our CIs :-/

https://github.com/predict-idlab/plotly-resampler/blob/c60833eb4382b3072cb591b36de737dbf769a5ea/pyproject.toml#L101

Reply:
OK. The `.so` file is created without the need for any extra calls, I can see it in the logs. Poetry-core does trigger its build. However, it is not making into the final package, not sure why.